chapters/shaders.adoc

```diff
@@ -2498,6 +2498,86 @@ In other shader stages, each invocation in a subgroup must: be in the same
 
 Only <<limits-subgroup-supportedStages, shader stages that support subgroup
 operations>> have defined subgroups.
+
+[NOTE]
+.Note
+====
+In shaders, there are two kinds of uniformity that is of interest to
+applications, subgroup uniform and dynamically uniform.
```

Comment:
```suggestion
In shaders, there are two kinds of uniformity that are of interest to
applications: subgroup uniform and dynamically uniform.
```

Reply:
Are there other types that are *not* of interest to apps?